### ISSUE TITLE
Unable to build release 2.3.0

### ISSUE THREAD

**Issue by aymanbagabas** [OzymandiasTheGreat/emoji-keyboard#31]
I'm getting this error with release 2.3.0

`
Failed to list sessions: Access denied
Traceback (most recent call last):
  File "setup.py", line 5, in <module>
    from lib.emoji_shared import version
  File "/tmp/src/emoji-keyboard-2.3.0/lib/emoji_shared.py", line 99, in <module>
    wayland = check_wayland()
  File "/tmp/src/emoji-keyboard-2.3.0/lib/emoji_shared.py", line 84, in check_wayland
    ['loginctl', 'show-session', session, '-p', 'Type', '--value'],
UnboundLocalError: local variable 'session' referenced before assignment
`

I solved it with by initiating session at the beginning [session.txt](https://github.com/OzymandiasTheGreat/emoji-keyboard/files/1823870/session.txt)

**Comment by OzymandiasTheGreat** [comment]
This seems to be a permission issue.
You don't seem to have the permission to list sessions, as to why - I have no idea.
A google search doesn't seem to turn up anything of relevance either.

**Comment by aymanbagabas** [comment]
Well, I was able to execute "loginctl list-sessions" which gave me c2 as the session name, then "loginctl show-session c2 -p Type --value" returns x11 which is the expected value. Therefore, I don't think it is a permission issue. However, after I initiated the variable in emoji_shared, everything worked perfectly.
What I did is, I cloned the repo then I did "sudo python3 setup.py install" and that is how I got all this.
I am running Archlinux with python 3.6.4 if that might help.
